### DISCUSSION TITLE
line to line fault.

### DISCUSSION THREAD

**Discussion by Adarsh-365** [CURENT/andes#503] [Q&A] [answered]
I am working on faults in grid can any one suggest how can i perform line-to-line fault.  from example, i get how to perform line-to-ground Fault.

**Comment by jinningwang** [comment, marked as answer]
There is no line-to-line fault model in ANDES, as it is a phasor-domain simulation where three-phase imbalance is ignored.
However, Fortescue might be helpful if you'd like to see line-to-line fault, https://docs.andes.app/en/latest/groupdoc/Interface.html#fortescue
